lit-next.code-workspace

```diff
@@ -58,7 +58,7 @@
     }
   ],
   "settings": {
-    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.tsdk": "lit-html/node_modules/typescript/lib",
```

Comment:
Debugging?

Reply:
This is just so I don't see errors in VS code that aren't there when building due to the difference TS versions. If we can figure out the renaming issue, this I would upgrade the rest of the repo. This wouldn't be merged as-is.